tests/plugins/os/windows/test_network.py

```diff
@@ -0,0 +1,322 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime
+from unittest.mock import patch
+
+import pytest
+
+from dissect.target.plugins.os.windows.network import WindowsNetworkPlugin
+from dissect.target.target import Target
+
+
+@dataclass
+class MockRegVal:
+    name: str
+    value: str | int
+
+
+class MockRegistry:
```

Comment:
Can you take a note out of other plugins instead? This is a bit overkill. While nice, it's best of this was implemented as something general usable for all unit tests, not just this one. For now I'd stay stick with the (arguably more annoying and worse) registry mapping.